ml-agents/mlagents/trainers/tests/test_subprocess_env_manager.py

```diff
@@ -147,7 +147,7 @@ def test_advance(self, mock_create_worker, external_brains_mock, step_mock):
         env_manager.set_agent_manager(brain_name, agent_manager_mock)
 
         step_info_dict = {brain_name: Mock()}
-        step_info = EnvironmentStep(step_info_dict, 0, action_info_dict)
+        step_info = EnvironmentStep(step_info_dict, 0, action_info_dict, {})
```

Comment:
We probably should put something in the dictionary so that the method gets tested, and similarly run `set_environment_stats` in the `test_agent_processor.py` tests

Reply:
Done